### ISSUE TITLE
"Snapshot to Resume" dropdown never populates

### ISSUE THREAD

**Issue by mikebind** [d8ahazard/sd_dreambooth_extension#869]
### Kindly read the entire form below and fill it out with the requested information.

**Please find the following lines in the console and paste them below. _If you do not provide this information, your
issue will be automatically closed._**

`
Python 3.10.6 | packaged by conda-forge | (main, Oct 24 2022, 16:02:16) [MSC v.1916 64 bit (AMD64)]
Commit hash: 6cff4401824299a983c8e13424018efc347b4a2b
Installing requirements for Web UI
[auto-sd-paint-ext] Commit hash: ff82c1dcf9309138d8f5c07546ca26e46ea8c3b7
[auto-sd-paint-ext] Auto-update disabled.

Python 3.10.6 | packaged by conda-forge | (main, Oct 24 2022, 16:02:16) [MSC v.1916 64 bit (AMD64)]
Commit hash: 6cff4401824299a983c8e13424018efc347b4a2b
Installing requirements for Web UI
[auto-sd-paint-ext] Commit hash: ff82c1dcf9309138d8f5c07546ca26e46ea8c3b7
[auto-sd-paint-ext] Auto-update disabled.

#######################################################################################################
Initializing Dreambooth
If submitting an issue on github, please provide the below text for debugging purposes:

Python revision: 3.10.6 | packaged by conda-forge | (main, Oct 24 2022, 16:02:16) [MSC v.1916 64 bit (AMD64)]
Dreambooth revision: 9f4d931a319056c537d24669cb950d146d1537b0
SD-WebUI revision: 6cff4401824299a983c8e13424018efc347b4a2b

Checking Dreambooth requirements...
[+] bitsandbytes version 0.35.0 installed.
[+] diffusers version 0.10.2 installed.
[+] transformers version 4.25.1 installed.
[+] xformers version 0.0.14.dev0 installed.
[+] torch version 1.13.1+cu116 installed.
[+] torchvision version 0.14.1+cu116 installed.

#######################################################################################################

Launching Web UI with arguments: --xformers --api --theme dark --autolaunch
`

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?**
Yes
**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?**
Yes
**Have you updated Dreambooth to the latest revision?**
Yes
**Have you updated the Stable-Diffusion-WebUI to the latest version?**
Yes
**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this.
Reply 'OK' Below to acknowledge that you did this.**
OK
**Describe the bug**

(A clear and concise description of what the bug is)
In the Dreambooth Tab, In the Model Selection area, the "Snapshot to Resume" dropdown list is never populated. There are no error messages, there is just nothing to select.  I think the use case for this is if you have a dreambooth checkpoint which has overtrained and you want to resume from an earlier point but reduce the learning rate, you would select the model from the "Model" dropdown and then select the earlier saved step from the "Snapshot to Resume", load the settings, then modify the learning rate (or whatever other settings), save the settings, and then click "Train" to start from the intermediate checkpoint.  There's nothing about this in the "readme" so I am just guessing that this is the intention of the "Snapshot to Resume" section, but maybe that's my misunderstanding...

**Provide logs**

If a crash has occurred, please provide the *entire* stack trace from the log, including the last few log messages *
before* the crash occurred.

**Environment**

What OS?
Windows 11
If Windows - WSL or native?
native
What GPU are you using?
RTX 3090 with 24GB
**Screenshots/Config**
If the issue is specific to an error while training, please provide a screenshot of training parameters or the
db_config.json file from /models/dreambooth/MODELNAME/db_config.json
Not a specific configuration I'm having trouble with, this dropdown appears to never be populated.

**Comment by d8ahazard** [comment]
Are you actually saving any snapshots of the diffusion weights? 

They are disabled by default, as each snapshot consumes ~6GB. Enable the box to save snapshots as you like, and as they are saved...they will be shown in the dropdown.

**Comment by mikebind** [comment]
I'm still trying to get the hang of parameters and how much to train, so yes, I do usually save multiple checkpoints, evaluate them afterwards, and delete all but the best one(s).    However, I gather from your comment that "snapshots" refers to diffusers snapshots (rather than ckpt files), and I do leave those unchecked (because I know how to use ckpt files and don't need another copy in the diffusers format for any reason I know of).  I guess if I wanted to resume from an intermediate ckpt file I could just use that as the base ckpt when creating a new model on the "Create" tab?  Any downside to doing it that way?
